modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -1199,12 +1199,51 @@ public void testBuildTemplateModulesExtGradle() throws Exception {
 		}
 	}
 
-	@Test(expected = Error.class)
+	@Test
 	public void testBuildTemplateModulesExtMaven() throws Exception {
-		_buildTemplateWithMaven(
-			"modules-ext", "foo-ext", "com.test",
-			"-DoriginalModuleName=com.liferay.login.web",
-			"-DoriginalModuleVersion=2.0.4");
+		File destinationDir = temporaryFolder.newFolder("maven");
```

Comment:
This line can be moved just before `_executeMaven` to follow the `as used` principle